docs/user/README.md

```diff
@@ -1,14 +1,19 @@
 The main interface to rust-analyzer is the
-[LSP](https://microsoft.github.io/language-server-protocol/) implementation. To
-install lsp server, use `cargo xtask install --server`, which is a shorthand for `cargo
-install --package ra_lsp_server`. The binary is named `ra_lsp_server`, you
```

Comment:
right, `install` does not support the `--package` flag, good catch! We should use `--path ./crates/ra_lsp_server` then instead. It is important that we specify the "desugard" way to install the thing, which doesn't rely on `xtask` magic.